test/functional/mining_prioritisetransaction.py

```diff
@@ -24,10 +24,7 @@
 class PrioritiseTransactionTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 1
-        self.extra_args = [[
-            "-printpriority=1",
-            "-datacarriersize=100000",
-        ]] * self.num_nodes
+        self.extra_args = [["-datacarriersize=100000"]] * self.num_nodes
```

Comment:
My bad, I assumed that since nothing was printed it was omitted, but
```
    if (m_print_modified_fee) {
        throw std::runtime_error("m_print_modified_fee is not implemented");
    }
```
fails - thanks for pointing it out.

Reply:
I've fixed the other Sonarcould warning in a separate commit, if you think it's better that way, I'll merge it with the first one: https://github.com/bitcoin/bitcoin/pull/30324/commits/71753aca8535764f077dc737e92784d2e56f40ef